KSP-Backend/KSP/mvp/models.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+from django.db import models
+
+YEAR_CHOICES = []
+for year in range(2016, (datetime.datetime.now().year + 1)):
+    YEAR_CHOICES.append((year, year))
+
+
+class Notes(models.Model):
+    course_id = models.CharField(max_length=10)
+    batch = models.IntegerField(choices=YEAR_CHOICES)
+    course_instructor = models.CharField(max_length=100)
+    uploaded_at = models.DateTimeField(auto_now=True)
+    gdrive_link = models.URLField(max_length=500)
+
+class Feedback(models.Model):
+    name = models.CharField(max_length=100)
+    course_id = models.CharField(max_length=10)
+    course_instructor = models.CharField(max_length=100)
+    batch = models.IntegerField(choices=YEAR_CHOICES)
+    created_date = models.DateTimeField(auto_now_add=True)
+    updated_date = models.DateTimeField(auto_now=True)
+    feedback = models.TextField()
+
+class PreviousYearQuestions(models.Model):
+    course_id = models.CharField(max_length=10)
+    batch = models.IntegerField(choices=YEAR_CHOICES)
+    course_instructor = models.CharField(max_length=100)
+    uploaded_at = models.DateTimeField(auto_now=True)
+    gdrive_link = models.URLField(max_length=500)
+
+class Resources(models.Model):

```

Comment:
Again this is exactly similar to Notes. Use notes model for all types of notes.

Reply:
I did this as it will be easier to retrieve the data later.

Reply:
Why not use one more field that can have one of the three values(notes, resource, previous year question) instead of creating three different models of the same schema? We can then filter the results based on this field.